### ISSUE TITLE
关于yolo_training.py中get_target函数的理解

### ISSUE THREAD

**Issue by Shaosifan** [bubbliiiing/yolov5-pytorch#14]
`            #-------------------------------------------------------#
            #   batch_target            : num_true_box, 4
            #   anchors                 : 9, 2
            #
            #   ratios_of_gt_anchors    : num_true_box, 9, 2
            #   ratios_of_anchors_gt    : num_true_box, 9, 2
            #
            #   ratios                  : num_true_box, 9, 4
            #   max_ratios              : num_true_box, 9
            #-------------------------------------------------------#
            ratios_of_gt_anchors = torch.unsqueeze(batch_target[:, 2:4], 1) / torch.unsqueeze(torch.FloatTensor(anchors), 0)
            ratios_of_anchors_gt = torch.unsqueeze(torch.FloatTensor(anchors), 0) /  torch.unsqueeze(batch_target[:, 2:4], 1)
            ratios               = torch.cat([ratios_of_gt_anchors, ratios_of_anchors_gt], dim = -1)
            max_ratios, _        = torch.max(ratios, dim = -1)
`
请问大家yolo_training.py，get_target函数中上面这段代码怎么理解啊，为什么要这样做了，把这个函数看了几遍，还是不太理解，求助！

**Comment by bubbliiiing** [comment]
进行先验框与真实框宽高的比对
